### PR TITLE
"Fixes" predator stun durations

### PR DESCRIPTION
**Pull request by InsaneRed**
# About the pull request
(this is up for debate from the council / pred players)

After fira fixed the code it actually works which has resulted in predators gettiing up from stuns way faster then ""intended"" im using intended because it was balanced around a system which didnt work, and now that it works they get up "faster"
# Explain why it's good for the game

predators are already busted to fight against as a xenomorph and their only counter (stuns) not working properly is not fun to play against

# Testing Photographs and Procedure

<details>

https://github.com/cmss13-devs/cmss13/assets/47158596/0226dd1e-0a7b-4cf6-8f9b-f873f28616d6

https://github.com/cmss13-devs/cmss13/assets/47158596/bc16e62f-924e-40a6-a969-a83840867db7

https://github.com/cmss13-devs/cmss13/assets/47158596/3af3f5e1-faea-4ae8-bc34-e162cb2238b5

^^^^ NEW
 ↓ ↓ ↓ OLD

https://github.com/cmss13-devs/cmss13/assets/47158596/6755d095-81fd-40f9-89c2-89532cb1a50e

https://github.com/cmss13-devs/cmss13/assets/47158596/66327438-4e2c-45fa-bc2b-bafe041e2607

https://github.com/cmss13-devs/cmss13/assets/47158596/3b9fad3b-559a-4a74-9968-0f818a27ac04

</details>

# Changelog

:cl:
balance: Properly tweaks predator stun resist back to what it should have been.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/yautja/_species.dm [hunk 1]
```diff
@@ -53,8 +53,8 @@
 		/mob/living/carbon/human/proc/mark_panel,
 	)
 
-	knock_down_reduction = 4
-	stun_reduction = 4
+	knock_down_reduction = 1.3
+	stun_reduction = 1.3
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/human/species/yautja/_species.dm:57]:
```
/mob/living/carbon/human/GetKnockDownDuration(amount)
	. = ..()
	var/skill_resistance = skills ? (skills.get_skill_level(SKILL_ENDURANCE)-1)*0.08 : 0
	var/final_reduction = (1 - skill_resistance) / species.knock_down_reduction
	return . * final_reduction
```
1.3 is a weird choice why not 1 or 2

Comment (ItsVyzo) [reply code/modules/mob/living/carbon/human/species/yautja/_species.dm:57]: It's easier to achieve proper balance when you use decimals

Comment (InsaneRed) [reply code/modules/mob/living/carbon/human/species/yautja/_species.dm:57]: because 1 was too little and 2 was too fast

Comment (InsaneRed) [reply code/modules/mob/living/carbon/human/species/yautja/_species.dm]: 1.3 works perfectly to how it was before thats why i settled on it, 1.5 is also fine

**Comment by Zonespace27** [comment]
Pretty equal to old times.

**Pull request merged**